__tests__/database/schema/UserSchema.test.ts

```diff
@@ -495,7 +496,8 @@ describe('TESTING UserSchema', () => {
         countryCode: 'countryCode',
         createdAt: 'createdAt',
         lastActiveAt: 'lastActiveAt',
-        profilePicture: 'profilePicture',
+        profilePicture:
+          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Hardcoding URLs directly into the codebase can lead to maintenance issues if the URL changes in the future. Consider storing this default profile picture URL in a configuration file or environment variable.
```diff
-        profilePicture:
-          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+        profilePicture: process.env.DEFAULT_PROFILE_PICTURE_URL,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->